ktxspec.adoc

```diff
@@ -849,8 +854,9 @@ value must be 0.
 An array, `levels`, giving the offset from the start of the file and
 compressed and uncompressed byte sizes of the image data for each
 mip level within the <<_mip_level_array,_Mip Level Array_>> The array is ordered
-starting with stem:[level_{base}] (the level with the largest size images)
-at index _0_. Image for stem:[level_p] will be found at index _p_.
+starting with stem:[\textit{level}_{base}] (the level with the largest size images)
+at index stem:[0]. The image for stem:[|textit{level}_p] will be found at index
```

Comment:
Should be `\` instead of `|`.

Reply:
Will make a separate PR for the Vulkan references.

I disagree with this change

```
stem:[\texttt{levels[}\textit{num_levels}-1\texttt{\].byteOffset}]
```

and similar. I think the current way with the skinny bracket looks better. The first "levels ..." on the first line in this image is with your suggested change. The second is as it is now.

<img width="918" alt="Screenshot 2025-02-17 at 16 41 10" src="https://github.com/user-attachments/assets/36cc0596-f30b-4dae-a58b-d80d46482d76" />

The other issues are resolved in https://github.com/KhronosGroup/KTX-Specification/pull/227/commits/f37f20c3bdd5a13b8b6dedb75efb29ebfceaafd3.

Reply:
> I think the current way with the skinny bracket looks better.

That's fair. My main concern is the extra gap after the dot. How about this:
```
stem:[\texttt{levels}[\textit{num_levels}-1\]\texttt{.byteOffset}]
```

Reply:
It doesn't seem to make much difference to the gap after the dot. It makes the spaces on both sides of the dot equal though. Is that what you want? The first expression has the change. The second is the original.

<img width="786" alt="Screenshot 2025-02-17 at 21 42 15" src="https://github.com/user-attachments/assets/42a59d7b-aab2-44bf-a13e-5acf40a7f136" />

Reply:
Regarding the Vulkan references, the registry main page now has only a single link to "latest" which points to a doc with the core spec. + extensions. I think I should remove the version number from the Vulkan references and also change the `vkFormat` section which has separate links to both core and extension. Both 1.2 links are redirected to "latest" by the Vulkan registry.

Off topic for this PR but I want to agree direction before I make a PR to update the links.

Reply:
Maybe I should link to https://docs.vulkan.org/spec/latest/chapters/formats.html for the reference in the `vkFormat` section. What do you think? I think the main Vulkan reference should be to the single html spec. document in the registry.

Reply:
Agreed on using the new chapter link for `VkFormat` and the "latest" for everything else.

Reply:
I just updated to Asciidoctor 2.0.23. I was previously using 2.0.20. I see very little difference. I even put copies of old and new on separate lines to see if that changed anything.

<img width="822" alt="Screenshot 2025-02-17 at 22 32 48" src="https://github.com/user-attachments/assets/46aa29da-a98d-4049-8e09-bf64cc4cafbf" />

Your rendering is different in a number of respects.  I wonder why.

I don't want to spend any more time on this so I will make the change.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Specification/pull/227/commits/4f7737a7e4b6b74b35a298772f5ed5dd80fe4817.